trunk/SUAVE/Methods/Aerodynamics/AVL/create_avl_datastructure.py

```diff
@@ -174,268 +174,215 @@ def populate_wing_sections(avl_wing,suave_wing):
+                                                sweep = suave_wing.Segments[i_segs].sweeps.leading_edge
+                                        else:          
+                                                sweep_quarter_chord = suave_wing.Segments[i_segs].sweeps.quarter_chord
+                                                chord_fraction      = 0.25 # quarter chord
+                                                segment_root_chord  = root_chord*suave_wing.Segments[i_segs].root_chord_percent
+                                                segment_tip_chord   = root_chord*suave_wing.Segments[i_segs+1].root_chord_percent
+                                                segment_span        = semispan*(suave_wing.Segments[i_segs+1].percent_span_location - suave_wing.Segments[i_segs].percent_span_location )
+                                                sweep               = np.arctan(((segment_root_chord*chord_fraction) + (np.tan(sweep_quarter_chord )*segment_span - chord_fraction*segment_tip_chord)) /segment_span)
+
+                                dihedral       = suave_wing.Segments[i_segs].dihedral_outboard
+
+                                # create a vector if all the sections to be made in each segment
+                                section_spans = []
+                                for cs in suave_wing.Segments[i_segs].control_surfaces:
+                                        control_surface_start = semispan*cs.span_fraction[0]
+                                        control_surface_end   = semispan*cs.span_fraction[1]
+                                        section_spans.append(control_surface_start)
+                                        section_spans.append(control_surface_end)
+
+                                # sort the section_spans in order to create sections in chronological order 
+                                ordered_section_spans = sorted(list(set(section_spans))) 
+
+                                # count the number of sections that the segment will contain
+                                num_sections = len(ordered_section_spans)       
+
+                                # create and append sections onto avl wing structure  
+                                for section_count in xrange(num_sections-1):
+                                        section                   = Section ()
+                                        section.tag               = suave_wing.Segments[i_segs].tag + '_section_at'+ str(ordered_section_spans[section_count]) + '_m'
+                                        root_section_chord        = root_chord*suave_wing.Segments[i_segs-1].root_chord_percent
+                                        tip_section_chord         = root_chord*suave_wing.Segments[i_segs].root_chord_percent
+                                        semispan_section_fraction = (ordered_section_spans[section_count] - semispan*suave_wing.Segments[i_segs-1].percent_span_location) /(semispan*segment_percent_span)  #check!!!  
```

Comment:
What's this "check" comment about?

Reply:
This was just for me to check when I was verifying that the wing geometry was translated correctly. I deleted the comment